Header header

float32 boomV
float32 armV
float32 boomP
float32 armP
int8 BoomMode
int8 ArmMode
